core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -668,6 +688,35 @@ oppia.directive('conversationSkin', [function() {
           return $scope.windowWidth >= TWO_CARD_THRESHOLD_PX;
         };
 
+        $scope.isNarrowViewport = function() {
+          return $scope.windowWidth < TWO_CARD_THRESHOLD_PX;
+        };
+
+        $scope.isTutorCardDisplayedOnNarrow = function() {
+          if (!$scope.isCurrentSupplementalCardNonempty()) {
+            return $scope.isNarrowViewport();
+          }
+          return $scope.isNarrowViewport() &&
+                 $scope._tutorCardDisplayedOnNarrow;
+        };
+
+        $scope.isSupplementalCardDisplayedOnNarrow = function() {
```

Comment:
Maybe: isScreenNarrowAndShowingSupplementalCard

(otherwise it sounds like 'narrow' here is a pre-condition, rather than something we're checking for)